tModPorter/tModPorter.Tests/TestData/FontsTest.Expected.cs

```diff
@@ -0,0 +1,17 @@
+using Terraria.ModLoader;
```

Comment:
we'll probably leave the unnecessary `using Terraria;`, as it's easy for modders to cleanup later, and figuring out whether it's now redundant is a bit of a pain.

Reply:
I'd rather not leave them unless the tester/rewriter breaks because of it

Reply:
The test framework will complain that the file produced by tModPorter (which won't remove the using) is different from the Expected.cs

Reply:
Anyway, is stuff I can tweak in 20s, so no worries